src/fn/fn-link.scss

```diff
@@ -0,0 +1,31 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-link;
+
+.#{$block} {
+  @include fn-reset();
+
+  color: $fn-color-blue-500;
+  text-decoration: none;
+
+  @include fn-hover() {
+    text-decoration: underline;
+  }
+
+  @include fn-focus() {
+    text-decoration: underline;
+  }
```

Comment:
Your focus is not correct. You need to write code for the following cases:
(You can check Button implementation)
```
@include fn-is-focus() { ... }
@include fn-not-focus-visible() { ... }
@include fn-focus-visible() { ... }
```
<img width="910" alt="Screen Shot 2021-12-10 at 11 08 03 AM" src="https://user-images.githubusercontent.com/39598672/145605101-d1fa0c82-1062-4244-99be-13f65afa5157.png">

Focus state does not have underline.